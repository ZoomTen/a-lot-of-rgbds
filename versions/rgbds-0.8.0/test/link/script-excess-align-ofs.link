rom0
	align 1, 2
