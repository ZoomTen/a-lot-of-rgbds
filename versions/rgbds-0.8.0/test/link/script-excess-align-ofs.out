error: script-excess-align-ofs.link(2): Cannot align: The alignment offset (2) must be less than alignment size (2)
Linking failed with 1 error
